modules/admin-ui/pom.xml

```diff
@@ -370,9 +370,8 @@
                   <goal>install-node-and-npm</goal>
                 </goals>
                 <configuration>
-                  <nodeVersion>v8.1.4</nodeVersion>
-                  <!-- npm 5.3.0 issue https://github.com/npm/npm/issues/17858 -->
```

Comment:
@karendolan IIRC, it was you that encountered problems with NPM 5.3.0. Could you please check whether those problems still occur on your develop environment?